fbfmaproom/tests/test_fbfmaproom.py

```diff
@@ -76,21 +77,33 @@ def test_generate_tables():
             '29.53', '39.90', '27.96', '28.82', '34.68', '35.84', '31.13',
             '35.42', '38.82', '38.87', '36.09'
         ],
-        obs_rank=[
+        rain_rank=[
             24., 33., 12., 38.,  8., 32., 16., 13., 35., 15.,  3., 37.,  5.,
             4., 19., 21., 20.,  9., 14., 10., 11.,  6.,  7., 18., 22., 30.,
             31., 25., 26.,  2., 28., 27., 34., 23., 39., 29., 36.,  1., 17.
         ],
+        ndvi_rank=[17.0, 21.0, 4.0, 20.0, 5.0, 15.0, 16.0, 8.0, 18.0, 7.0, 1.0,
+                   19.0, 3.0, 2.0, 14.0, 12.0, 13.0, 10.0, 9.0, 11.0, 6.0,
+                   np.nan, np.nan, np.nan, np.nan, np.nan, np.nan, np.nan,
+                   np.nan, np.nan, np.nan, np.nan, np.nan, np.nan, np.nan,
+                   np.nan, np.nan, np.nan, np.nan
+        ],
         bad_year=[
             '', '', '', '', 'Bad', 'Bad', '', '', '', '', 'Bad',
             '', 'Bad', 'Bad', '', '', 'Bad', '', '', '', '', 'Bad',
             'Bad', '', '', '', '', '', '', 'Bad', '', '', '',
             '', '', '', '', 'Bad', ''
         ],
-        worst_obs=[
+        worst_rain=[
             0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 1, 1, 0, 0, 0, 1, 0, 1, 1, 1,
             1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 0
         ],
+        worst_ndvi=[
+            0.,  0.,  1.,  0.,  1.,  0.,  0.,  0.,  0.,  0.,  1.,  0.,  1.,  1.,
+            0.,  0.,  0.,  0., 0.,  0.,  1., np.nan, np.nan, np.nan, np.nan,
+            np.nan, np.nan, np.nan, np.nan, np.nan, np.nan, np.nan, np.nan,
+            np.nan, np.nan, np.nan, np.nan, np.nan, np.nan
+        ],
```

Comment:
This is kind of ugly--if the column has NaNs in it, it gets cast to float instead of int, because int doesn't have a missing value marker. I don't really want to enshrine this behavior in a test, but I decided to do the expedient thing for now because I plan to eliminate the worst_* columns soon enough.